src/components/ChartComponent/ChartComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Line } from "react-chartjs-2";
+import { Spin } from "antd";
+import "./ChartComponent.scss"
+
+export const ChartComponent: React.FC<any> = (chartData) => {
+
+    return (
+        <div className="chart-container">
+            {!!chartData ?
```

Comment:
Spinner should be visible only when data is fetching. If there will be No data reserved, then we should display empty view instead od loader, otherwise user can wait for some data needlessly

Reply:
Will do that Radzio